packages/react-topology/package.json

```diff
@@ -31,14 +31,28 @@
     "@patternfly/react-core": "^4.12.2",
     "@patternfly/react-icons": "^4.2.7",
     "@patternfly/react-styles": "^4.2.5",
-    "tslib": "^1.11.1"
+    "tslib": "^1.11.1",
+    "mobx": "^5.14.2",
+    "mobx-react": "^6.1.4",
+    "mobx-react-lite": "^1.4.2",
+    "point-in-svg-path":"^1.0.1",
+    "react-measure": "^2.3.0",
+    "d3":"^5.16.0"
   },
   "peerDependencies": {
     "react": "^16.8.0",
-    "react-dom": "^16.8.0"
+    "react-dom": "^16.8.0",
+    "d3":"^5.16.0",
+    "@types/dagre": "0.7.42",
+    "@types/webcola": "3.2.0",
+    "dagre": "0.8.2",
+    "webcola": "3.4.0"
```

Comment:
These are already dev deps. Why not make them normal deps instead of dev and peer deps?

Reply:
done